tests/Feature/Models/UserTest.php

```diff
@@ -61,7 +61,7 @@ public function getFullNameDataProvider(): array
     public function testGetFullName(array $attributes, string $fullName)
     {
         // Given
-        $user = new User($attributes);
+        $user = new AuthorizedUser($attributes);
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = new AuthorizedUser($attributes);
````